ksqldb-rest-app/src/test/java/io/confluent/ksql/rest/entity/QueryDescriptionFactoryTest.java

```diff
@@ -117,7 +117,7 @@ public void setUp() {
         queryCloseCallback,
         closeTimeout);
 
-    transientQueryDescription = QueryDescriptionFactory.forQueryMetadata(transientQuery);
+    transientQueryDescription = QueryDescriptionFactory.forQueryMetadata(transientQuery, Collections.emptyMap());
```

Comment:
The passed map should always be empty for now since the only way forQueryMetadata() would be called on a transient query would be in ExplainExecutor.explainStatement() and the query isn't actually running at that point

Reply:
after https://github.com/confluentinc/ksql/issues/4308 is completed, there should eb tests with non-empty maps for transient queries